cmds/ps/ps.go

```diff
@@ -177,8 +177,8 @@ func ps(pT ProcessTable) error {
 	sort.Sort(pT)
 
 	if flags.x {
-		pT.headers = []string{"PID", "TTY", "STAT", "TIME", "COMMAND"}
-		pT.fields = []string{"Pid", "Ctty", "State", "Time", "Cmd"}
+		pT.headers = []string{"PID", "PGRP", "SID", "TTY", "STAT", "TIME", "COMMAND"}
+		pT.fields = []string{"Pid", "Pgrp", "Sid", "Ctty", "State", "Time", "Cmd"}
```

Comment:
Do you want those fields in cases without flag.x too? It seems to me that you should